src/snowflake/snowpark/modin/plugin/compiler/snowflake_query_compiler.py

```diff
@@ -10069,6 +10069,24 @@ def dt_property(self, property_name: str) -> "SnowflakeQueryCompiler":
                     pandas_lit(False),
                 )
             ),
+            "is_quarter_start": (
+                lambda column: coalesce(
+                    (dayofmonth(col(column)) == 1)
+                    & (month(col(column)).in_(1, 4, 7, 10)),
```

Comment:
let's make this list into a constant variable with a name like QUARTER_START_MONTHS.

Reply:
Done.